src/render/quantization.cpp

```diff
@@ -176,8 +201,11 @@ Image* convert_pixel_format(
 
             if (a == 0)
               *dst_it = new_mask_color;
-            else if (rgbmap)
-              *dst_it = rgbmap->mapColor(r, g, b, a);
+            if (octreeMap)
+                *dst_it = octreeMap->mapColor(r, g, b, a, new_mask_color);
```

Comment:
Remove extra indentation:
```suggestion
              *dst_it = octreeMap->mapColor(r, g, b, a, new_mask_color);
```